opentelemetry-etw-logs/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "opentelemetry-etw-logs"
+description = "OpenTelemetry-Rust exporter to ETW"
```

Comment:
nit - Should we change it to - `OpenTelemetry logs exporter to ETW (Event Tracing for Windows)`? In consistent with metrics exporter - https://github.com/open-telemetry/opentelemetry-rust-contrib/blob/eb0422adbd93c42b1c17ec4578aa020efae25b4a/opentelemetry-etw-metrics/Cargo.toml#L5C16-L5C81